benefit/data/sheets/ComplexTypes/ComplexTypes.json

```diff
@@ -380,7 +380,7 @@
   {"LiveFrom": "18/06/2019", "ID": "hearingRecordingRequest", "ListElementCode": "status", "FieldType": "Text", "ElementLabel": "Status", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/01/2018", "ID": "hearingRecordingRequest", "ListElementCode": "dateRequested", "FieldType": "Date", "ElementLabel": "Date requested", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/01/2018", "ID": "hearingRecordingRequest", "ListElementCode": "requestDocument", "FieldType": "Document", "ElementLabel": "Document", "SecurityClassification": "PUBLIC"},
-  {"LiveFrom": "18/06/2019", "ID": "hearingRecordingRequest", "ListElementCode": "sscsHearingRecordingList", "FieldType": "Collection", "ElementLabel": "Hearing recordings", "FieldTypeParameter": "sscsHearingRecordingDetails", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "18/06/2019", "ID": "hearingRecordingRequest", "ListElementCode": "sscsHearingRecording", "FieldType": "sscsHearingRecordingDetails", "ElementLabel": "Hearing recording", "SecurityClassification": "PUBLIC"},
```

Comment:
Why we have fieldType to a specific field name?

Reply:
Don't we need to specify the type of the field, same like "FieldType": "Text" and "FieldType": "Document" etc?

and line number 353
  {"LiveFrom": "01/01/2018", "ID": "reasonableAdjustments", "ListElementCode": "jointParty", "FieldType": "reasonableAdjustmentDetails", "ElementLabel": "Joint Party", "SecurityClassification": "PUBLIC"},

It is also reffer to reasonable adjustment type